openapi/api.github.com/operations/repos/list-topics.json

```diff
@@ -15,7 +15,8 @@
       "schema": {
         "type": "string",
         "default": "application/vnd.github.v3+json"
```

Comment:
The `schema.default` should be set to the required preview header, compare https://github.com/octokit/routes/blob/e035b1064dab83fe2b35cd0abac9931057880307/openapi/api.github.com/operations/apps/get-authenticated.json#L10-L21

Reply:
oh good call, doing this fixes all the issues with speccy as well! 
<img width="848" alt="image" src="https://user-images.githubusercontent.com/2539016/67740303-89b06d00-f9d2-11e9-8a5b-9970ff2bc1f9.png">